Assets/Scripts/UI/Core/UIDictElement.cs

```diff
@@ -0,0 +1,49 @@
+﻿/*
+ * Author(s): Isaiah Mann
+ * Description: Sets UI text based of dict lookup (based on language)
+ * Usage: [no notes]
+ */
+
+public abstract class UIDictElement : UIElement 
+{	
+	#region Instance Accessors
+
+	public bool HasLanguage
+	{
+		get 
+		{
+			return language != null;
+		}
+	}
+
+	#endregion
+
+	LanguageDatabase language;
```

Comment:
Not at the moment because the only access we need is provided by the public method `string GetTerm(string key)`